tests/Resources/ti.media.test.js

```diff
@@ -38,6 +38,37 @@ describe('Titanium.Media', function () {
 		should(Ti.Media.previewImage).be.a.Function;
 	});
 
+	it.android('previewImage of non-file blob', function () {
```

Comment:
my bad, I moved this test into the suite so there's a conflict now. You can move this to a ti.media.addontest.js or copy the ti/media.test front he suite and add this (or uncomment the variant of this I wrote using takeScreenshot which also exposed the issue).